backend/src/main/java/ai/verta/modeldb/experiment/ExperimentDAORdbImpl.java

```diff
@@ -208,10 +208,11 @@ public Experiment insertExperiment(Experiment experiment, UserInfo userInfo)
     createRoleBindingsForExperiment(experiment, userInfo);
     try (Session session = modelDBHibernateUtil.getSessionFactory().openSession()) {
       Transaction transaction = session.beginTransaction();
-      session.save(RdbmsUtils.generateExperimentEntity(experiment));
+      ExperimentEntity experimentEntity = RdbmsUtils.generateExperimentEntity(experiment);
+      session.save(experimentEntity);
```

Comment:
return experiment without sorting fields so need to call toProtos for sorting fields